src/test/java/org/jabref/gui/exporter/SaveDatabaseActionTest.java

```diff
@@ -0,0 +1,103 @@
+package org.jabref.gui.exporter;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import org.jabref.gui.BasePanel;
+import org.jabref.gui.DialogService;
+import org.jabref.gui.JabRefFrame;
+import org.jabref.gui.util.FileDialogConfiguration;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.database.shared.DatabaseLocation;
+import org.jabref.preferences.JabRefPreferences;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class SaveDatabaseActionTest {
```

Comment:
Travis architecture error message might be related to the fact that the TestClass is public.
Remove the public modifier. At least I remember that I once had the same problem and fixed it